tableau_utilities/scripts/apply_configs.py

```diff
@@ -0,0 +1,281 @@
+from copy import deepcopy
+import pprint
+from typing import Dict, Any, List
+from time import time
+
+
+from tableau_utilities.tableau_file.tableau_file import Datasource
+from tableau_utilities.general.cli_styling import Color, Symbol
+from tableau_utilities.general.config_column_persona import personas
+from tableau_utilities.scripts.datasource import add_metadata_records_as_columns, create_column
+from tableau_utilities.scripts.gen_config import build_configs
+from tableau_utilities.scripts.merge_config import read_file
+
+color = Color()
+symbol = Symbol()
```

Comment:
```suggestion
COLOR = Color()
SYMBOL = Symbol()
```
By convention, I would make these uppercase since they'll be using globally in this script